payment_bacen_pix/readme/CREDITS.rst

```diff
@@ -0,0 +1,6 @@
+[ This file is optional and contains additional credits, other than
+  authors, contributors, and maintainers. ]
```

Comment:
tirar essas duas linhas acima. Do restante a principio a gente costuma destacar quem financiou quando é diferente do autor, mas enfim se quiser ficar forçando a barra sempre vc é quem sabe a reputação que vc constrói na comunidade.